pkg/models/trb_request_attendee.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// TRBRequestAttendee represents an EUA user who is included as an attendee for a TRB request
+type TRBRequestAttendee struct {
+	ID           uuid.UUID  `json:"id"`
```

Comment:
this should embed base struct instead of implementing metadata fields directly. (See TRBRequest as an example).